fms/models/llama.py

```diff
@@ -418,21 +431,27 @@ def load_fms_llama(model_path: str, group=None, **kwargs):
     return ibm_model
 
 
-def convert_hf_llama(hf_model: "LlamaForCausalLM") -> LLaMA:
+def convert_hf_llama(hf_model: "LlamaForCausalLM", ignore_weights=False, orig_init=False) -> LLaMA:
     """
     Convert a Llama huggingface model to an fms model
 
     Parameters
     ----------
     hf_model: LlamaForCausalLM
         a Llama Huggingface model
+    ignore_weights: bool
+        If True, load config but use random weights
```

Comment:
If the goal is to get a randomly initialized instance of the LLaMA class, the user can just call `get_model('llama', '7b')` for example and it will be randomly initialized. I'm not sure if there's a good use-case for this option here